src/Poseidon/Utils.hs

```diff
@@ -64,7 +65,7 @@ renderPoseidonException PoseidonEmptyForgeException =
     "Nothing to be forged"
 renderPoseidonException (PoseidonNewPackageConstructionException s) =
     show s
-
+renderPoseidonException (PoseidonGenericException s) = show s
```

Comment:
The show here is superfluous, I believe. `s` is already a String. Wrapping it in `show` adds unnecessary quotes:
```
[clemens@cranberry data_ancient]$ trident list --individuals -d . -j Country -j GroupName --ignoreGeno
Searching POSEIDON.yml files... 134 found
Initializing packages... 134 initialized
Packages loaded: 134
Preparing output table
"GroupName is not a valid Janno column name"
```

Reply:
Completely correct. I'll commit a patch